packages/dx-react-grid/demos/featured/controlled-mode.md

```diff
@@ -1,6 +1,5 @@
 # React Grid Controlled State Mode
 
-This demo shows the Grid capability to delegate its UI state management to the parent component. You can contol either the whole Grid state or just a part of it. In the
-first case, Grid becomes a stateless component. This mode is usefull if you need to use the Grid state in other parts of your application. Using this mode, you can also persist the component state and restore it when required.
+This demo demonstrates the use of the Grid in the [Controlled State mode](/devextreme-reactive/react/grid/docs/guides/controlled-and-uncontrolled-modes/). That is the parent component manages the whole Grid's state or just its part. In the first case, the Grid works as a stateless component. Use the Controlled State mode when you need to share the Grid's state among other application parts. It also allows you to persist the Grid's state and restore it when required.
```

Comment:
I am not sure that we should use relative URL here. It seems that we should use links to the `.md` files like we do in the `docs` folder.